### ISSUE TITLE
[terra-brand-footer]

### ISSUE THREAD

**Issue by mjhenkes** [cerner/terra-framework#1357]
Update the terra-brand-footer package to indicate that it will be deprecated in the future.
Update terra-brand-footer documentation to indicate that it will be deprecated in the future using the deprecation notice.

This component will be supported by terra until a broad terra change that would require a major version. (e.g. React 17).

With the introduction of the application framework via terra-application, the brand footer no-longer has a place in the layout and the information contained within should be located elsewhere.

Per audit #1209

**Comment by nickpith** [comment]
👍
